### PR TITLE
Update Direct Debits icon

### PR DESCRIPTION
**Pull request by hendrikmolder**
## ❓ Context
 
We've updated Direct Debits icons. These changes are already reflected in Figma.

## 🚀 Changes

- Renames previous `direct-debits` icon to `calendar`.
- Adds new `direct-debits` icon.

## 💬 Considerations

[Thread](https://transferwise.slack.com/archives/CJNBX9CP6/p1617711373084700?thread_ts=1617706025.084300&cid=CJNBX9CP6)

## ✅ Checklist

- [x] Make the PR title meaningful
- [x] The package version is bumped according to README.md in `package.json`, and `CHANGELOG.md`

### CODE REVIEW

**Review by oliverviljamaa — APPROVED** [review]
Is this double-checked with designers?

**Comment by gusso** [comment]
Small problem on the direct debit exports from Figma, you need to select the containers that are sized `24x24` and `16x16` rather than the vector layer inside, to maintain their expected size. Always good to double check on the SVG after exporting them to see if it contains the properties `width="24" height="24"` or `width="16" height="16"`.

Screenshot 2021-04-06 at 1 42 57 pm

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+# v2.7.0
+
+## Update Direct Debits icon
+
+- Previous `direct-debits` icon is renamed to `calendar`.
+- Adds new `direct-debits` icon.
```
Comment (dozortsev) [inline CHANGELOG.md:6]: this looks like visual breaking change, I might think of publishing it as breaking change 🤔

Comment (hendrikmolder) [reply CHANGELOG.md:6]: I was thinking that too – but the aim is to use the correct `direct-debit` icon across the product, so even though it's visually breaking/different icon, I don't think we're breaking anything for consumers of `direct-debit` icon 🤔

File: src/icons/direct-debits/outline_16.svg [hunk 2]
```diff
@@ -1,4 +1,5 @@
-<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path d="M9.5 11C9.5 10.1716 10.1716 9.5 11 9.5C11.8284 9.5 12.5 10.1716 12.5 11C12.5 11.8284 11.8284 12.5 11 12.5C10.1716 12.5 9.5 11.8284 9.5 11Z" fill="#00BAFF"/>
-<path fill-rule="evenodd" clip-rule="evenodd" d="M12.8 2.19995H15.8V15.8H0.199951V2.19995H2.70005V0H4.30005V2.19995H11.2V0H12.8V2.19995ZM1.79995 14.2V6H14.2V14.2H1.79995ZM11.2 3V4.5H12.8V3H11.2ZM2.70005 3V4.5H4.30005V3H2.70005Z" fill="#00BAFF"/>
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="none">
```
Comment (hendrikmolder) [inline src/icons/direct-debits/outline_16.svg:1]: @gusso I've fixed the exporting issue you mentioned – thanks 🙌 

But I noticed I'm missing `viewBox` property – do I need to add that?

File: CHANGELOG.md (hunk 1 above)
Comment (dozortsev) [reply CHANGELOG.md:6]: alright, I see, make sense 👍

File: src/icons/direct-debits/outline_16.svg (hunk 2 above)
Comment (gusso) [reply src/icons/direct-debits/outline_16.svg:1]: Hmm, I wonder why we're getting different results. My export has the `viewBox` attribute, and it doesn't have the empty `<defs/`> element. Can you share a screenshot of your Figma right before you click on the export button?

File: CHANGELOG.md (hunk 1 above)
Comment (oliverviljamaa) [reply CHANGELOG.md:6]: I agree — it's not something _to look out for_, it's actually aligning the with the intended behaviour.

File: src/icons/direct-debits/outline_16.svg (hunk 2 above)
Comment (hendrikmolder) [reply src/icons/direct-debits/outline_16.svg:1]: I ran them through svgomg, maybe that's why 😬  I guess that's not really needed here, so will use the ones straight from Figma 👍

Comment (dozortsev) [reply src/icons/direct-debits/outline_16.svg]: @hendrikmolder yeah this is not necessary, we clean up and optimize SVG as part of `yarn build` 🙂

**Review by dozortsev — APPROVED** [review]

**Pull request merged**